apps/prairielearn/src/pages/instructorCourseAdminInstances/instructorCourseAdminInstances.html.ts

```diff
@@ -16,11 +21,33 @@ export function InstructorCourseAdminInstances({
   resLocals: Record<string, any>;
   courseInstances: CourseInstanceAuthzRow[];
 }) {
+  const startDate = Temporal.Now.zonedDateTimeISO(resLocals.course.timeZone).with({
+    hour: 0,
+    minute: 1,
+    second: 0,
+  });
+  const initialStartDateFormatted = formatDateYMDHM(
+    new Date(startDate.epochMilliseconds),
+    resLocals.course.time_zone,
+  );
+
+  const endDate = startDate.add({ months: 4 }).with({
+    hour: 23,
+    minute: 59,
+    second: 0,
+  });
```

Comment:
We use these defaults in a few other places. @mwest1066 any preference here?

```suggestion
  const startDate = Temporal.Now.zonedDateTimeISO(resLocals.course.timeZone).with({
    hour: 0,
    minute: 0,
    second: 1,
  });
  const initialStartDateFormatted = formatDateYMDHM(
    new Date(startDate.epochMilliseconds),
    resLocals.course.time_zone,
  );

  const endDate = startDate.add({ months: 4 }).with({
    hour: 23,
    minute: 59,
    second: 59,
  });
```

Reply:
We've previously used 00:00:01 and 23:59:59 so that it's basically midnight from either side. However, I think we rarely (if ever) want seconds level of precision in our times, and it adds a lot of visual noise throughout the interface. PT has consistently used 00:01 and 23:59 with date inputs set to use minutes as the step size, and we haven't heard any complaints. I'm wondering whether we can do the same in PL? The only problem I see is one of backwards compatibility with pre-existing dates in access rules.

Reply:
Hmm, interesting! Certainly we could do something where we only display seconds for editing if they're non-zero. This may not even apply to this PR, where we're always creating things from scratch, so if you want to move in the direction of minute precision by disabling seconds here, I'd be OK with that.